packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1010,3 +1010,124 @@ authfactor.challenge.soft_token.used_passcode = Each code can only be used once.
 # rate limit
 oie.tooManyRequests = Too many requests
 E0000010 = Server is unable to respond at the moment.
+
+# OIDC/enduser consent scopes
+consent.scopes.openid.label=openid
+consent.scopes.email.label=View your email address.
+consent.scopes.profile.label=View your profile information.
+consent.scopes.phone.label=View your phone number.
+consent.scopes.address.label=View your address.
+consent.scopes.offlineAccess.label=Keep you signed in to the app.
+consent.scopes.onlineAccess.label=Keep you signed in to the app.
+consent.scopes.deviceSSO.label=Request device secret.
+consent.scopes.openid.desc=Signals that a request is an OpenID request.
+consent.scopes.profile.desc=The exact data varies based on what profile information you have provided, such as: name, time zone, picture, or birthday.
+consent.scopes.email.desc=This allows the app to view your email address.
+consent.scopes.address.desc=This allows the app to view your address, such as: street address, city, state, and zip code.
+consent.scopes.phone.desc=This allows the app to view your phone number.
+consent.scopes.offlineAccess.desc=This keeps you signed in to the app, even when you are not using it.
+consent.scopes.onlineAccess.desc=This allows the app to access your data only when you are logged in to the application.
+consent.scopes.deviceSSO.desc=This allows you to single sign on between native client applications on your device.
+# Admin consent scopes
+consent.scopes.okta.users.manage.label=okta.users.manage
```

Comment:
why these needs to be localized ? `okta.users.manage`.. Was that the case in v1 ?

Reply:
Shouldn't every string besides "names" be localized? It's not localized in v1.

Reply:
This is like an event/variable name and should not be localized